docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md

```diff
@@ -0,0 +1,90 @@
+
+To activate a product:
+1. Navigate to the product variant of the product that you want to activate:
+    **Edit** > **Variants** > **Edit product variant**
+2.  In the top right corner of the *Edit Concrete Product* page, click **Activate**.
+The product turns visible to the customers of your online store.
+
+{% info_block infoBox "Note" %}
+
+Each variant needs to be activated in order to be visible to your customers.
+
+{% endinfo_block %}
+
+**Tips & tricks**
+<br>If at some point in time you want to hide the product variant from your customers, you just deactivate it using the same procedure described above. This deactivates only the product variant. The abstract product is active until at least one its variant is active.
+
+## Viewing products
+
+If you need to review the product details without actually editing them, do the following:
+1. In the _Actions_ column of the abstract product you want to view, click **View**.
+2. On the *View Product* page, you can navigate to the view product variant, initiate the editing flow for it, or manage its attributes.
+
+**Tips & tricks**
+<br>If you notice something you would like to change for your product, in the top right corner of the page. click **Edit**.
+
+## Managing product attributes
+
+When you see the **Manage Attributes** option, keep in mind that you manage the attributes like *brand* but not the super attributes. Such attributes like brand do not define the product variants differentiation, meaning they are not used while defining the concrete products of an abstract product. They rather go to the details section on the product details page in your online store. You can manage attributes for both abstract and concrete products.
+
+{% info_block infoBox "Info" %}
+
+The attributes that you add are taken from the **Products** > **Attributes** section of the Back Office. So the attribute you want to define should exist in that section.
+
+{% endinfo_block %}
+
+To manage the product attributes:
+1.  In the _Actions_ column or in the top right corner of the *Edit* page, select the **Manage Attributes** option for the concrete or abstract product.
+2. On the *Manage Attributes for Product* page, type the first three letters of the attribute key.
+3. Select the suggested value and click **Add**.
+4. In the _Attributes_ section, define the **Default** value for your attributes and specify the value for the **locales**.
+    Repeat the procedure if needed.
+5. Click **Save**.
+
+See the _References_ section to see the examples of how the attributes look like.
+
+
+## Approving and denying the marketplace products
+
+Depending on the current status of the product, you can approve, deny or get the product in the *draft* status.
+To update the approval status of the product, in the _Actions_ column of the abstract product do the following:
+
+- For the product with status *Draft*:
+  - click **Send for Approval** to initiate the approval process.
+  - click **Approve** to approve the product.
```

Comment:
when it's a draft, there is no action buttons, it is waiting for the merchant to send for approval

Reply:
prevoius comment https://prnt.sc/sEZciJ_gqwqG